messages/bin_ltfs/root.txt

```diff
@@ -159,6 +159,7 @@ root:table {
 		14120W:string { "Cannot access to directory %s, disabled index capture mode (%d)." }
 		14121I:string { "Index will be captured at %s at update" }
 		14122I:string { "Index will not be captured." }
+		14123W:string { "Fuse main returned the error code: (%d)." }
```

Comment:
`:` is little bit funny to me. 

It shall be `The main function of the FUSE returned error (%d).`

Reply:
Hello Abe-San
Thanks for the observation. I changed it to use "The main function of FUSE returned error (%d)." instead.

Please let me know if you have any other comment. 

Regards.

Reply:
Looks good to me.